microsoft-edge/devtools-guide-chromium/crash-analyzer/index.md

```diff
@@ -6,108 +6,108 @@ ms.author: msedgedevrel
+<!-- ------------------------------ -->
+#### Format the stack traces for the Crash analyzer tool
 
-Produce a specially-formatted stack trace that includes a section called `Source modules`.  The `Source modules` section contains the source map references to the JavaScript functions that were part of the error stack trace.  You can produce such a stack trace by using the [Crash Analyzer Support](https://www.npmjs.com/package/@microsoft/edge-devtools-crash-analyzer-support) npm package.
+Collecting stack traces in production is not enough. The **Crash analyzer** tool needs stack traces to include a section called **Source modules**.
 
-To install the Crash Analyzer Support npm package, at a command prompt, such as git bash, enter the install command that's shown at the above page:
+Here is an example of a stack trace that includes the **Source modules** section:
 
 ```
-npm i @microsoft/edge-devtools-crash-analyzer-support
+RangeError: Maximum call stack size exceeded
+    at new q (https://example.com/bundled/entrypoints/heap_snapshot_worker/heap_snapshot_worker.js:1:38478)
+    at K.createEdge(https://example.com/bundled/entrypoints/heap_snapshot_worker/heap_snapshot_worker.js:1:32931)
+    at G.edge (https://example.com/bundled/entrypoints/heap_snapshot_worker/heap_snapshot_worker.js:1:6912)
+    at G.name (https://example.com/bundled/entrypoints/heap_snapshot_worker/heap_snapshot_worker.js:1:6267)
+    at f (https://example.com/bundled/entrypoints/heap_snapshot_worker/heap_snapshot_worker.js:1:30931)
+    at https://example.com/bundled/entrypoints/heap_snapshot_worker/heap_snapshot_worker.js:1:31513
+    at https://example.com/bundled/core/platform/platform.js:1:330
+    at r (https://example.com/bundled/core/platform/platform.js:1:376)
+    at r (https://example.com/bundled/core/platform/platform.js:1:425)
+    at r (https://example.com/bundled/core/platform/platform.js:1:425)
+
+Source modules:
+    https://example.com/bundled/entrypoints/heap_snapshot_worker/heap_snapshot_worker.js 9e8af998e1e1bbdb3ede85775d2c64825ccaff4b835c8b4238f10952b3890144
+    https://example.com/bundled/core/platform/platform.js 9a35dc3d31ba75643a6248e885e91caf800e4a293284695d1e96ab519dc563b2
 ```
 
+The **Source modules** section includes the URLs of the JavaScript files that involved in the stack trace, and a hash of the content of each file.  The hash is a 64-character string of hexadecimal characters corresponding to the SHA-256 hash of the script being executed.  This is a byte-for-byte hash of the content for each JavaScript file.  The URLs and hashes later make it possible for the **Crash analyzer** tool to retrieve the source maps used to unminify the stack trace.
```

Comment:
i'll probably insert "that are" used to unminify -- per corp-level style guide advise

Reply:
The URLs and hashes make it possible for the **Crash analyzer** tool to later retrieve the source maps that were used to unminify the stack trace.